### PR TITLE
Hmis 956 dev

### PR DESCRIPTION
**Pull request by subrotomukherjee**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/HMIS-960

### Change description ###
Adding end point to create linked hearing group

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/test/java/uk/gov/hmcts/futurehearings/hmi/unit/testing/testsuites/POST_Resources_LinkedHearingGrp_UnitTests.java [hunk 1]
```diff
@@ -0,0 +1,221 @@
...
+        headersAsMap.put("Request-Type", "THEFT");
+        headersAsMap.put("Request-Created-At", "2018-01-29T20:36:01Z");
+    }
+
+    @Test
+    @Order(1)
+    @DisplayName("Test for Invalid Resource for linked-hearing-group")
+    void testCreateLinkedHearingGroupResourceForInvalidResource() throws IOException {
+        final String input = givenAPayload(CORRECT_CREATE_LINKED_HEARING_GROUP_RESOURCE_PAYLOAD);
+        final Response response = httpClient.httpPost(resourcesApiRootContext+"/linkedHearingGroup"+"post", headersAsMap, paramsAsMap, input);
```
Comment (Matthew-Summerbell) [inline src/test/java/uk/gov/hmcts/futurehearings/hmi/unit/testing/testsuites/POST_Resources_LinkedHearingGrp_UnitTests.java:115]: Create a variable?

File: src/test/java/uk/gov/hmcts/futurehearings/hmi/unit/testing/testsuites/POST_Resources_LinkedHearingGrp_UnitTests.java [hunk 2]
```diff
@@ -0,0 +1,221 @@
...
+        final Response response = createResource(input);
+        thenValidateResponseForMissingOrInvalidAccessToken(response);
+    }
+
+    private Response createResourceNoAuth(final String input) {
+        return httpClient.httpPostNoAuth(resourcesApiRootContext+"/linked-hearing-group", headersAsMap, paramsAsMap, input);
+    }
+
+    private Response createResource(final String input) {
+        return httpClient.httpPost(resourcesApiRootContext+"/linked-hearing-group", headersAsMap, paramsAsMap, input);
```
Comment (Matthew-Summerbell) [inline src/test/java/uk/gov/hmcts/futurehearings/hmi/unit/testing/testsuites/POST_Resources_LinkedHearingGrp_UnitTests.java:215]: create a variable?

File: src/test/java/uk/gov/hmcts/futurehearings/hmi/unit/testing/testsuites/POST_Resources_LinkedHearingGrp_UnitTests.java [hunk 3]
```diff
@@ -0,0 +1,221 @@
...
+    void testCreateLinkedHearingGroupResourceWithInvalidAccessToken() throws IOException {
+        accessToken = TestUtilities.getToken(grantType, invalidClientID, invalidClientSecret, invalidTokenURL, invalidScope);
+        httpClient.setAccessToken(accessToken);
+        final String input = givenAPayload(CORRECT_CREATE_LINKED_HEARING_GROUP_RESOURCE_PAYLOAD);
+        final Response response = createResource(input);
+        thenValidateResponseForMissingOrInvalidAccessToken(response);
+    }
+
+    private Response createResourceNoAuth(final String input) {
+        return httpClient.httpPostNoAuth(resourcesApiRootContext+"/linked-hearing-group", headersAsMap, paramsAsMap, input);
```
Comment (Matthew-Summerbell) [inline src/test/java/uk/gov/hmcts/futurehearings/hmi/unit/testing/testsuites/POST_Resources_LinkedHearingGrp_UnitTests.java:211]: create a variable?

Comment (subrotomukherjee) [reply src/test/java/uk/gov/hmcts/futurehearings/hmi/unit/testing/testsuites/POST_Resources_LinkedHearingGrp_UnitTests.java]: done

File: src/test/java/uk/gov/hmcts/futurehearings/hmi/unit/testing/testsuites/POST_Resources_LinkedHearingGrp_UnitTests.java (hunk 2 above)
Comment (subrotomukherjee) [reply src/test/java/uk/gov/hmcts/futurehearings/hmi/unit/testing/testsuites/POST_Resources_LinkedHearingGrp_UnitTests.java]: done

File: src/test/java/uk/gov/hmcts/futurehearings/hmi/unit/testing/testsuites/POST_Resources_LinkedHearingGrp_UnitTests.java (hunk 1 above)
Comment (subrotomukherjee) [reply src/test/java/uk/gov/hmcts/futurehearings/hmi/unit/testing/testsuites/POST_Resources_LinkedHearingGrp_UnitTests.java]: done

**Review by mjotichmcts — APPROVED** [review]

**Pull request merged**